app/src/main/kotlin/io/intrepid/skotlinton/base/BaseActivity.kt

```diff
@@ -23,7 +23,7 @@ abstract class BaseActivity : AppCompatActivity() {
     }
 
     @CallSuper
-    override fun onNewIntent(intent: Intent?) {
+    override fun onNewIntent(intent: Intent) {
```

Comment:
Pretty confident this can't actually be null (how would the system know a null intent should be delivered to our app?) but let me know if I'm mistaken